### PR TITLE
Using lapack fortran handles

### PR DESCRIPTION
**Pull request by ItsPitt**

### CODE REVIEW

**Comment by ItsPitt** [comment]
Ready for first review

File: Makefile [hunk 1]
```diff
@@ -153,42 +153,38 @@ endif
 # --- Main target variable definitions ----------------------------------------
 #
 
-MK_BASE_FLAMEC_SRC                    :=
-MK_BASE_FLAMEC_OBJS                   :=
+MK_BASE_FLAMEC_SRC                          :=
+MK_BASE_FLAMEC_OBJS                         :=
 
-MK_BLAS_FLAMEC_SRC                    :=
```
Comment (iotamudelta) [inline Makefile]: Can you fix those whitespace  changes?

File: build/post-configure.sh.in [hunk 2]
```diff
@@ -50,6 +50,7 @@ echo "Enable built-in BLAS implementation............. : @fla_enable_builtin_bla
 echo ""
 echo "Enable lapack2flame............................. : @fla_enable_lapack2flame@"
 echo "Enable lapack2flash (experimental).............. : @fla_enable_lapack2flash@"
```
Comment (iotamudelta) [inline build/post-configure.sh.in:52]: Can you add documentation for the new legacy LAPACK option?

File: configure [hunk 3]
```diff
@@ -1528,6 +1530,13 @@ Optional Features:
                           ENABLED while external-lapack-for-subproblems and
                           lapack2flame MUST be DISABLED. (Disabled by
                           default.)
+  --enable-legacy-lapack  Compile and build into libflame the legacy f2c files
+                          of lapack (Version 3.5.0). If this option is not
```
Comment (iotamudelta) [inline configure]: -> "of LAPACK (Version 3.5.0)"

And similar below - in text LAPACK should be all caps.

**Review by iotamudelta — CHANGES_REQUESTED** [review]
Only small comments.

File: Makefile (hunk 1 above)
Comment (ItsPitt) [reply Makefile]: Fixed:  https://github.com/iotamudelta/libflame/pull/4/commits/959b92826d2a5e688f739ac96c83d73139158a39

File: configure (hunk 3 above)
Comment (ItsPitt) [reply configure]: Fixed: https://github.com/iotamudelta/libflame/pull/4/commits/959b92826d2a5e688f739ac96c83d73139158a39

File: Makefile (hunk 1 above)
Comment (iotamudelta) [reply Makefile]: looks like there is still a single whitespace off.....

File: build/post-configure.sh.in (hunk 2 above)
Comment (ItsPitt) [reply build/post-configure.sh.in:52]: Fixed: https://github.com/iotamudelta/libflame/pull/4/commits/6c46c7ba244bcde2c0c4896f2cd9ed9b70aee18b

File: Makefile (hunk 1 above)
Comment (ItsPitt) [reply Makefile]: fixed? https://github.com/iotamudelta/libflame/pull/4/commits/6c46c7ba244bcde2c0c4896f2cd9ed9b70aee18b

**Review by iotamudelta — APPROVED** [review]
LGTM for now.

**Pull request merged**